lib/Workflow/Persister/DBI/SequenceId.pm

```diff
@@ -13,14 +13,12 @@ $Workflow::Persister::DBI::SequenceId::VERSION = '1.52';
 my @FIELDS = qw( sequence_name sequence_select );
 __PACKAGE__->mk_accessors(@FIELDS);
 
-my ($log);
```

Comment:
With the removal of this statement, I cannot see where the log handle comes from in this class, it does not inherit from any classes provided the log handle via the main object - or am I missing something?

Reply:
As this is the same pattern, using the same solution. (I've replaced this now with a `log` accessor in the class which is initialized upon instantiation.)